### DISCUSSION TITLE
[Timeseries] Add TimeRegions support

### DISCUSSION THREAD

**Discussion by simPod** [grafana/grafana#49032] [Dashboards, Panels & Visualizations]
TimeRegions feature from Graph is missing in Timeseries panel.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
